package.json

```diff
@@ -562,6 +562,12 @@
           "command": "mssql.editTable",
           "when": "view == objectExplorer && viewItem =~ /\\btype=(Table)\\b/ && config.mssql.enableRichExperiences"
         }
+      ],
+      "webview/context": [
+        {
+          "command": "mssql.copyAll",
+          "when": "webviewId == 'queryResult' && webviewSection == 'queryResultMessagesPane'"
```

Comment:
Not including the rich feature flag here as the `queryResult` view inherently requires the flag to be enabled.